plugins/Diagnostics/Diagnostic/CronArchivingCheck.php

```diff
@@ -77,7 +77,8 @@ public function execute()
             }
             $comment .= $this->translator->translate('Installation_NotSupported')
                 . ' ' . $this->translator->translate('Goals_Optional')
-                . ' (' . $this->translator->translate('General_Reasons') . ': ' . $reasonText . ')';
+                . ' (' . $this->translator->translate('General_Reasons') . ': ' . $reasonText . ')'
+                . $this->translator->translate('General_LearnMore', [' <a target="_blank" href="https://matomo.org/faq/troubleshooting/how-to-make-the-diagnostic-managing-processes-via-cli-to-display-ok/">', '</a>']);
             $status = DiagnosticResult::STATUS_INFORMATIONAL;
```

Comment:
@sgiehl So, below, we still need to set `$escapeComment` to false, right?

Reply:
actually not, as it doesn't use the `informationalResult` method, but created the `DiagnosticResultItem` item directly

Reply:
:+1: